test/kata1.fizzBuzz.test.js

```diff
@@ -1,11 +1,22 @@
 const { fizzBuzz } = require("../src");
 
 describe("fizzBuzz", () => {
-  test("returns Fizz when passed a multiple of 3", () => {});
+  test("returns Fizz when passed a multiple of 3", () => {
+    expect(fizzBuzz(3)).toBe("Fizz");
```

Comment:
Always good to have at least a couple fo tests for each scenario to make sure this one isnt passing a a fluke